services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/PaymentMethod.java

```diff
@@ -0,0 +1,182 @@
+package com.mapbox.api.directions.v5.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.api.directions.v5.DirectionsAdapterFactory;
+
+/*
+ * Payment method for the toll road. See `TollCost`, `PaymentsMethods`.
+ *
+ * since 6.7.0
+ */
+@AutoValue
+public abstract class PaymentMethod extends DirectionsJsonObject {
```

Comment:
This class list different sizes of vehicles accepted and how much it costs for each different vehicle size. Let's change the name to `CostPerVehicleSize` as the data it hosts is not a `PaymentMethod`